src/workspacer/Keybinds/KeybindManager.cs

```diff
@@ -56,9 +57,38 @@ public KeybindManager(IConfigContext context)
             thread.Start();
         }
 
+        public void ShowKeybindWarning(string warningMessage)
+        {
+            if (_keybindWarning == null)
+            {
+                _keybindWarning = new TextBlockMessage("workspacer keybinds", "Warning, duplicate keybinds!", warningMessage, new List<Tuple<string, Action>>()
+                {
+                    new Tuple<string, Action>("ok", () => { }),
+                });
+            }
+
+            if (_keybindWarning.Visible)
+            {
+                _keybindWarning.Hide();
+            }
+            else
+            {
+                _keybindWarning.Show();
+            }
+        }
+
         public void Subscribe(KeyModifiers mod, Keys key, KeybindHandler handler, string name)
         {
             var sub = new Sub(mod, key);
+            var test = new LauncherResponse();
```

Comment:
As far as I can see, this test-variable is never used for anything. 

If so, it should be removed. If not, it needs to be documented with a comment or something why it needs to be here.

Reply:
correct was used for testing, should be removed